### PR TITLE
[Merged by Bors] - feat: MeasurableSpace (Fin n)

### PR DESCRIPTION
**Pull request by nomeata**
This defines the instance `MeasurableSpace (Fin n)`.
The motivation is #6425, which defines `Pmf.binomial`, and thus a measure on `Fin n`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/MeasurableSpace.lean [hunk 1]
```diff
@@ -388,6 +389,7 @@ instance Bool.instMeasurableSingletonClass : MeasurableSingletonClass Bool := 
 #align bool.measurable_singleton_class Bool.instMeasurableSingletonClass
 instance Nat.instMeasurableSingletonClass : MeasurableSingletonClass ℕ := ⟨fun _ => trivial⟩
 #align nat.measurable_singleton_class Nat.instMeasurableSingletonClass
+instance Fin.instMeasurableSingletonClass n : MeasurableSingletonClass (Fin n) := ⟨fun _ => trivial⟩
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/MeasurableSpace.lean:392]: I've never seen this syntax before: can we go with the clearer
```suggestion
instance Fin.instMeasurableSingletonClass (n : ℕ) : MeasurableSingletonClass (Fin n) := ⟨fun _ => trivial⟩
```
(and below)

Comment (nomeata) [reply Mathlib/MeasureTheory/MeasurableSpace.lean]: Sure!

**Comment by eric-wieser** [comment]
I'm not sure if the commit message is correct, c.f. https://github.com/leanprover-community/mathlib4/pull/6425/files#r1286449490

**Comment by eric-wieser** [comment]
I note this is not marked awaiting-review; are you waiting on a Zulip thread?

**Comment by nomeata** [comment]
> I note this is not marked awaiting-review; are you waiting on a Zulip thread?

I wasn’t sure initially, but it seem innocent enough I guess?

**Comment by nomeata** [comment]
I’ll be traveling without a laptop for week, and will address review comments afterwards. Should anyone want to adopt this PR and see it through, that’s fine with me as well.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**